models/intermediate/int_sage_intacct__general_ledger_balances.sql

```diff
@@ -175,12 +181,12 @@ final as (
             first_value(period_ending_amount_starter) over (partition by gl_partition order by period_last_day rows unbounded preceding)) as period_beg_amount,
         coalesce(period_ending_amount_starter,
             first_value(period_ending_amount_starter) over (partition by gl_partition order by period_last_day rows unbounded preceding)) as period_ending_amount
```

Comment:
I think we would still need this here since each `source_relation` has its won `gl_partition`.
```suggestion
            first_value(period_ending_amount_starter) over (partition by gl_partition{{ sage_intacct.partition_by_source_relation() }} order by period_last_day rows unbounded preceding)) as period_beg_amount,
        coalesce(period_ending_amount_starter,
            first_value(period_ending_amount_starter) over (partition by gl_partition{{ sage_intacct.partition_by_source_relation() }} order by period_last_day rows unbounded preceding)) as period_ending_amount
```